src/schema/partner_show.ts

```diff
@@ -93,9 +104,49 @@ const PartnerShowType = new GraphQLObjectType({
           ).then(({ body }) => body)
         }
 
+        // @ts-ignore
+        // FIXME: Update with Gravity param when ready
         return fetch.then(exclude(options.exclude, "id"))
       },
     },
+    artworksConnection: {
+      description: "A connection of artworks featured in the show",
+      type: artworkConnection,
+      args: pageable(artworksArgs),
+      resolve: (
+        show,
+        options,
+        _request,
+        { rootValue: { partnerShowArtworksLoader } }
+      ) => {
+        const loaderOptions = {
+          partner_id: show.partner.id,
+          show_id: show.id,
+        }
+
+        let gravityArgs = convertConnectionArgsToGravityArgs(options)
+
+        if (options.exclude) {
+          gravityArgs.exclude_ids = flatten([options.exclude])
+          delete gravityArgs.exclude
+        }
+
+        delete gravityArgs.page
+
+        return Promise.all([
```

Comment:
I'm confused why there is two calls here? Just to load artworks for a show requires the one call via `partnerShowArtworksLoader`. If you need the total count, you can get that from `show.eligible_artworks_count`, _or_ pass `total_count: true` to the artworks loader and resolve both `body`, and `headers['x-total-count']`.

Reply:
I copied this connection directly from partner, will update!